entity-service/config/detekt/baseline.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
```

Comment:
i didn't get it, it's used for what ?

Reply:
it is explained here: https://detekt.github.io/detekt/baseline.html

(we'll fix these existing code smells later, but at least, we won't add new ones)

Reply:
So having code smells does not break the build

Reply:
existing code smells won't break it (but we'll have to fix them), new ones will break it.